src/libs/DraftCommentUtils.ts

```diff
@@ -41,7 +41,7 @@ function hasValidDraftComment(reportID: string): boolean {
 function prepareDraftComment(comment: string | null) {
     // logical OR is used to convert empty string to null
```

Comment:
This can be removed now, right

Reply:
Thanks for pointing this out! I hurried up and left over nullish (`??`) instead of regular OR fallback here (`||`). The OR should stay here as we want to send `null` when empty string is provided to reset draft in the Onyx cache.